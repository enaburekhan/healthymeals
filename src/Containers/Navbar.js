import { Link } from 'react-router-dom';

const NavBar = () => (
  <div className="nav header">
    <Link className="home-link" to="/">
      <h4 className=" nav-text"> Healthy Meals</h4>
    </Link>
  </div>
);

export default NavBar;
