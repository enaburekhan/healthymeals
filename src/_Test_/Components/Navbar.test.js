import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Navbar from '../../Containers/Navbar';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
