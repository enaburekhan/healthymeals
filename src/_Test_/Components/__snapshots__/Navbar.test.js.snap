// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="nav header"
>
  <a
    className="home-link"
    href="/"
    onClick={[Function]}
  >
    <h4
      className=" nav-text"
    >
       Healthy Meals
    </h4>
  </a>
</div>
`;
